### PR TITLE
configured gatsby to create blog posts from .md files

### PR DESCRIPTION
**Pull request by lwbecker**
Blog now takes .md files from the `src/markdown-pages` directory and turns them into blog posts.

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
 const mainText = `This is the home page of the blog about Liam's summer internship at Viget!
   Each week there will be a new blog post about how things are going`;
 
-const IndexPage = () => {
+const IndexPage = ({
+    data: {
+        allMarkdownRemark: { edges },
+    },
+}) => {
+    const Posts = edges
```
Comment (averyvery) [inline src/pages/index.js:14]: Small style note: Typically `PascaleCase`-style variables are only used for classes and constructors functions in JS. So while `IndexPage` is correct, `Posts` would usually be lowercase – it's just an array of React nodes, not an actual class you could create a new instance of.

File: src/markdown-pages/post1.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+Thursday we had our final orientation meeting with [Brian Williams](https://www.viget.com/about/team/bwilliams), where he talked about Viget's history, starting all the way back in [Cindy](https://www.viget.com/about/team/ccaldwell)'s basement. Having recently celebrated its 20th birthday, Brian told us about how Viget has survived many hardships and is well established to survive the current economic downturn caused by the coronavirus pandemic.
+
+#### Project Pitch
+
+Today the group of interns pitched our idea for the group project that we'd like to work on during our 8-week tenure: a digital "shopping cart" app that allows the user to see the environmental impact of the groceries that they put into their cart.
+
+#### Shouout
+
+Lastly I'd like to give a shoutout to all my fellow interns Jenny Montoya (UX Research), Mika Byar (UX Design), Jack Doyle (Data & Analytics) and Will Dineen (JavaScript Developer). Here's to seven more weeks!
```
Comment (averyvery) [inline src/markdown-pages/post1.md:33]: :+1: this is a good post! I would correct "Shouout" to "Shoutout" and "Dineen" to "Dinneen", just two little things I noticed.

File: src/markdown-pages/post1.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+#### Viget Systems
+
+Wednesday we spoke with Viget President [Andy Rankin](https://www.viget.com/about/team/arankin) about Viget's business model, where he talked about the balance of overdelivering client work. There is a time and a place for overdelivery, but Andy stresses the importance of not going overboard. To explain why Viget can't always overdeliver, he compared them to a Honda salesman. If a Honda salseman were to sell a Honda but deliver a Maserati, the client would be exceptionally happy with their purchase and probably recommend that dealership to many more people. However, those new clients would then be disappointed if they bought a Honda and only recieved what they purchased. As with most good things, moderation is key.
```
Comment (averyvery) [inline src/markdown-pages/post1.md:21]: Not a code review comment but I'm always happy to hear that we're still using this car metaphor. 

Going beyond just clients, I think this industry is _full_ of people with money who think that money can buy much more than it really does, and handling their requests is a constant struggle. Neat article on the topic: http://kyleprifogle.com/dear-startup/

File: src/markdown-pages/post1.md (hunk 2 above)
Comment (lwbecker) [reply src/markdown-pages/post1.md]: Nice catch, fixed!

File: src/pages/index.js (hunk 1 above)
Comment (lwbecker) [reply src/pages/index.js]: Good to know, thank you! 

That was leftover from some of the boilerplate code that Gatsby provided for adding markdown pages, which adds another strange thing that they do. I wonder if they're consistent with that kind of thing like they are with the backticks? Something I may look in to!

**Pull request merged**